fastlane/Fastfile

```diff
@@ -385,9 +387,18 @@ platform :ios do
       derived_data_path: "scan_derived_data",
       output_types: '',
       fail_build: false,
-      result_bundle: true
+      result_bundle: true,
+      output_directory: "fastlane/test_output/xctest/ios"
     )
-    trainer(path: "fastlane/test_output", output_directory: "fastlane/test_output/xctest/ios")
+    remove_xcresults if is_ci
+  end
+
+  lane :remove_xcresults do
```

Comment:
Curious why this is needed? I think sometimes it's useful to look through the artifacts to find out more info.

Reply:
It was taking CircleCI like 20 minutes to upload the `.xcresult` 😬 

Before this _fastlane_ change the `xcresult` files were not included in the `test_output` folder (like they are now) so they were not getting uploaded as `store_test_results`. 

See [this run](https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/4952/workflows/e35ec051-a1d5-471a-b82b-12f485d99613/jobs/15643) from https://github.com/RevenueCat/purchases-ios/pull/1198 👇

<img width="1278" alt="Screen Shot 2022-01-20 at 12 52 16 PM" src="https://user-images.githubusercontent.com/401294/150403295-1096d8a0-2c2f-47fe-b1dd-eed70ee6fb20.png">
g uploaded to as an artifact. 

Previously `scan` put the `xcresult` in a temp directory which is where `trainer` was picking it up from. I'm working on making a change for this in `2.201.0.rc4` but I wanted to get this in here to make sure incase that didn't work out how I wanted and because I wanted to make sure CircleCI properly handled the junit files 😊 

If anything they we should replace the `xcresult` files somewhere that is `store_artifacts` but not `store_test_results`

Reply:
Follow up...

`scan` previously output `test_output/RevenueCat.xcresult` 
`trainer` previously output `test_output/xctest/iOS/RevenueCat.xml`

So the `store_test_results` did not include the `xcresult` but... because `scan` and `trainer` use the same `output_directory` now they do 😬 

Figuring out a way to separate

Reply:
@NachoSoto This should be resolved now 😊 Decided to zip the `xcresult` because `CircleCI` hates that they are actually directories